tools/ci/check_grep.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -euo pipefail
 
+## DO NOT ADD SPELLCHECKING TO THIS SCRIPT
+## NOTHING NEW SHOULD BE ADDED TO THIS
+## MAKE A BETTER REPLACEMENT
+## -ZephyrTFA
```

Comment:
What is the supposed alternative to spellchecking

Reply:
Don't say use a VScode addon, you're removing spellchecks which are not in normal dictionaries

Reply:
Don't have one.
Use your eyes.

Reply:
I will DNM until one exists that is better than Use Your Eyes

Reply:
Check grep was never intended to be used as spell check.
This is being removed.

Reply:
We are altering the deal